--- closure-library/closure/goog/ui/colorpicker.js	2020-04-03 16:06:00.288835500 +0200
+++ closure-libraryPatched/closure/goog/ui/colorpicker.js	2020-04-03 16:06:00.323835500 +0200
@@ -333,11 +333,11 @@
   // olives
   '#ffffcc', '#ffff33', '#ffff00', '#ffcc00', '#999900', '#666600', '#333300',
   // greens
-  '#99ff99', '#66ff99', '#33ff33', '#33cc00', '#009900', '#006600', '#003300',
+  '#99ff99', '#66ff99', '#00ff00', '#33cc00', '#009900', '#006600', '#003300',
   // turquoises
   '#99ffff', '#33ffff', '#66cccc', '#00cccc', '#339999', '#336666', '#003333',
   // blues
-  '#ccffff', '#66ffff', '#33ccff', '#3366ff', '#3333ff', '#000099', '#000066',
+  '#ccffff', '#66ffff', '#0000ff', '#3366ff', '#3333ff', '#000099', '#000066',
   // purples
   '#ccccff', '#9999ff', '#6666cc', '#6633ff', '#6600cc', '#333399', '#330099',
   // violets
